src/algebraic_geometry/locally_ringed_space.lean

```diff
@@ -135,6 +135,13 @@ forget_to_SheafedSpace ⋙ SheafedSpace.forget _
 @[simp] lemma comp_val {X Y Z : LocallyRingedSpace} (f : X ⟶ Y) (g : Y ⟶ Z) :
   (f ≫ g).val = f.val ≫ g.val := rfl
 
+@[simp] lemma comp_val_c {X Y Z : LocallyRingedSpace} (f : X ⟶ Y) (g : Y ⟶ Z) :
+  (f ≫ g).val.c = g.val.c ≫ (presheaf.pushforward _ g.val.base).map f.val.c := rfl
```

Comment:
I don't really find the RHS much simpler than the LHS. Should this be in the default simp-set?

Reply:
I think this simp-lemma would be useful most of the times.